docs/demo-script.md

```
@@ -0,0 +1,333 @@
+
+### Flow
+
+1. Show `docker compose` running Postgres and supporting services.
+2. Show the weather data table or weather service cache.
+3. Show the remote A2A server command:
+
+```bash
+adk api_server --a2a --port 8001 stylist_agents/remote_a2a
+```
+
+4. Open Agent Cards:
```

markdownlint-cli2 (0.22.1)

[warning] Ordered list item prefix
Expected: 1; Actual: 4; Style: 1/1/1

(MD029, ol-prefix)